app/cinema/hall.py

```diff
@@ -0,0 +1,16 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie: str,
+                      customers: list,

```

Comment:
```suggestion
                      customers: list[Customer],
```
You can give more descriptive type annotation

Reply:
I agree